modules/andes-core/broker/src/main/java/org/wso2/andes/store/cassandra/CQLBasedAndesContextStoreImpl.java

```diff
@@ -195,6 +196,14 @@ private void createSchema() {
      * {@inheritDoc}
      */
     @Override
+    public Map<String, String> getAllDurableSubscriptionsByID() throws AndesException {
+        throw new NotImplementedException();
```

Comment:
Thanks for pointing out. CQLBasedAndesContextStoreImpl is removed since Cassandra is not supported anymore.